Marlin/Conditionals_LCD.h

```diff
@@ -92,7 +101,7 @@
     #endif
   #endif
 
-  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) || ENABLED(ANET_FULL_GRAPHICS_LCD)
```

Comment:
I agree - I've used the Anet Full Graphics display on my Anet A8 for months with only REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER enabled.

Keeping ANET_FULL_GRAPHICS_LCD just as an alias seems a bit wasteful to me - better just to drop it now, IMHO.

That said, if keeping it is preferred for backwards compatibility, something like this in Conditionals_LCD should take care of it:
```
#if ENABLED(ANET_FULL_GRAPHICS_LCD)
  #define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
#endif
```

The pins definitions are already in the pins file, and anyone who wants to use the RepRap Full Graphics display on their Anet should be directed towards the 'alternate' wiring adapter by @ralf-e , which doesn't require changing any of the pin definitions so the conditional block in pins_ANET10.h wouldn't be required any more either (the alternate version for the Thingiverse adapter could stay in the comments, or live on Thingiverse).